tests/NoUnused/ExportsTest.elm

```diff
@@ -1021,6 +1012,8 @@ main = text ""
 """
                 , """
 module OtherModule exposing (..)
+import Test
+a : Test.Test
```

Comment:
I'm guessing this is to prevent `a` from being reported?
If so, you should instead reference it in the `NotReported`, for instance by doing `main = OtherModule.a`.